style.css

```diff
@@ -0,0 +1,113 @@
+:root {
+  --bg-main-mobile: #1c1a19;
+  --bg-header: #3c3a39;
+  --body-bg-image: url("./images/illustration-header-mobile.png");
+}
+
+body {
+  height: 100vh;
+  min-width: 375px;
+  place-items: center;
+  overflow: hidden;
+}
```

Comment:
- [ ] It would be better to add a **CSS `*` Selector** and set all the padding by default to ```0`` and the margin also to make your content fix on the web page 😊😊

Example:
```
* {
padding: 0;
margin: 0;
}

Reply:
Great advice, thank you.
It's Done.